src/main/java/com/ecmsp/orderservice/order/config/OrderConfiguration.java

```diff
@@ -1,25 +1,31 @@
 package com.ecmsp.orderservice.order.config;
 
+import com.ecmsp.orderservice.order.adapter.generator.RandomOrderIdGenerator;
 import com.ecmsp.orderservice.order.domain.*;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 import java.time.Clock;
-import java.util.UUID;
 
 @Configuration
 class OrderConfiguration {
 
+    @Bean
+    RandomOrderIdGenerator orderIdGenerator(){
```

Comment:
Change returned type to interface type - `OrderIdGenerator` to hide implementation details